### PR TITLE
refactor: Refine procedure section

### PR DESCRIPTION
**Pull request by stu01509**
In this PR, I refine the Providers, Configure, Load Compendium, Pending Review, Patient Results, Batch Results, Electronic Reports, and Lab Documents pages.

#### Short description of what this resolves:
- Remove outdated code and inline-style.
- Convert the CSS to bootstrap classes.
- Adjust the bootstrap grid system.

##### Procedure > Providers:
image

image

##### Procedure > Configuration:
image

image

##### Procedure > Load Compendium:
image

##### Procedure >  Pending Review & Procedure > Patient Results & Procedure > Batch Results:
image

##### Procedure > Electronic Reports:
image

##### Procedure > Lab Documents:
image

### CODE REVIEW

File: interface/orders/list_reports.php [hunk 1]
```diff
@@ -166,404 +166,424 @@ function openPatient(pid) {
...
+                // Generate procedure columns.
+                if ($order_seq && $lastpcid != $order_seq) {
+                    if ($sendable) {
+                        echo "  <td>" . text($procedure_code) . "</td>\n";
+                        echo "  <td>" . text($procedure_name) . "</td>\n";
+                    } else {
+                        echo "  <td><strike>" . text($procedure_code) . "</strike></td>\n";
+                        echo "  <td><strike>" . text($procedure_name) . "</strike></td>\n";
+                    }
+                } else {
```
[the remark this answers is not in the document]
Comment (tywrenn) [reply interface/orders/list_reports.php:548]: This is deprecated. Replace with the `<s>` tag.

File: interface/main/display_documents.php [hunk 2]
```diff
@@ -143,114 +143,116 @@ function expandOrCollapse(type,prefixString) {
...
-        </span>
+<body>
+<div class="container mt-3">
+    <div class="row">
+        <div class="col-12">
+            <h2 class='title'><?php echo xlt('Lab Documents'); ?></h2>
+            <span id='docexpand' onclick='expandOrCollapse(1,"doc")' style='cursor:pointer;<?php echo $display_expand_msg ?>'>(<?php echo xlt('expand'); ?>)</span>
+            <span id='doccollapse' onclick='expandOrCollapse(2,"doc")' style='cursor:pointer;<?php echo $display_collapse_msg ?>'>(<?php echo xlt('collapse'); ?>)</span>
+            <br />
+            <br />
```
[the remark this answers is not in the document]
Comment (tywrenn) [reply interface/main/display_documents.php:161]: Yeah you can remove it. I don't think we need it unless @bradymiller says otherwise.

**Comment by stu01509** [comment]
Hi @tywrenn  and @bradymiller 

In Providers, Configuration, the add form looks like have nested form, so do you recommend me to remove it?
image

**Comment by tywrenn** [comment]
@stu01509 It appears to just have the card class tacked onto it. You can leave it alone for now.

**Comment by tywrenn** [comment]
@stu01509 Please rebase this

**Comment by stu01509** [comment]
@tywrenn  Done, please check again.

File: interface/orders/procedure_provider_edit.php [hunk 3]
```diff
@@ -70,8 +70,8 @@ function invalue($name)
 
 </head>
```
Comment (tywrenn) [inline interface/orders/procedure_provider_edit.php:72]: The ? symbol that appears when you hover over a field and allows you to click for more information does not work at all.
image

File: interface/orders/types_edit.php [hunk 4]
```diff
@@ -85,10 +85,9 @@ function recursiveDelete($typeid)
 <html>
```
Comment (tywrenn) [inline interface/orders/types_edit.php:85]: The ? icon tooltip here does nothing

File: interface/orders/types.php [hunk 5]
```diff
@@ -63,7 +63,7 @@
 <html>
```
Comment (tywrenn) [inline interface/orders/types.php:63]: The ? tooltip here does nothing

File: interface/orders/list_reports.php [hunk 6]
```diff
@@ -166,404 +166,424 @@ function openPatient(pid) {
 </script>
```
Comment (tywrenn) [inline interface/orders/list_reports.php:166]: Where do I trigger this area...

File: interface/orders/types.php (hunk 5 above)
Comment (stu01509) [reply interface/orders/types.php:63]: The tooltip in the `td` tag has a little bit issue when I mouse enter on the tooltip, the tooltip will flash.
https://www.loom.com/share/66a9c9e129b24828a00b3834c391e3ff

File: interface/orders/procedure_provider_edit.php (hunk 3 above)
Comment (stu01509) [reply interface/orders/procedure_provider_edit.php:72]: Update the commit.
image

File: interface/orders/orders_results.php [hunk 7]
```diff
@@ -169,39 +169,6 @@ function QuotedOrNull($fld)
     font-size: 13px;
 }
```
[the remark this answers is not in the document]
Comment (stu01509) [reply interface/orders/orders_results.php:171]: Update the commit.
image

File: interface/orders/types_edit.php (hunk 4 above)
Comment (stu01509) [reply interface/orders/types_edit.php:85]: Update the commit.
image

File: interface/orders/list_reports.php (hunk 6 above)
Comment (stu01509) [reply interface/orders/list_reports.php:166]: In Procedures > Electronic Reports

I guess it needs to import the HL7 file. 

cc @bradymiller, @stephenwaite

File: interface/orders/types.php (hunk 5 above)
Comment (sjpadgett) [reply interface/orders/types.php:63]: .tooltip { 
    pointer-events: none;
}

Comment (stu01509) [reply interface/orders/types.php:63]: I fix the flashing issue but seems like the tooltip position still has some issue, I try to set the placement but it still looks like the same.

image

Comment (sjpadgett) [reply interface/orders/types.php:63]: whats the tips css position set to, relative, fixed or something else. simple way to chaeck is rt click and inspect.

Comment (stu01509) [reply interface/orders/types.php:63]: Hi @tywrenn @bradymiller  and @sjpadgett 

Seems like the tooltip in `table` `td` its position will be a move to on top of the table, I write some example, please take a look.
https://codepen.io/CliffSu/pen/WNwXJQQ?editors=1010

Comment (sjpadgett) [reply interface/orders/types.php:63]: hi @stu01509 try data attr `data-placement="right"` or bottom. ya may have to add a position relative or fixed.

**Comment by stu01509** [comment]
Update the commit, set the tooltip position to the right, look good!
Thanks @sjpadgett 

image

**Review by tywrenn — APPROVED** [review]
@bradymiller

File: interface/batchcom/test.js [hunk 8]
```diff
@@ -0,0 +1,4 @@
+for (var index = 0; index < array.length; index++) {
```
Comment (bradymiller) [inline interface/batchcom/test.js:1]: What is this file meant for?

File: interface/orders/list_reports.php [hunk 9]
```diff
@@ -166,404 +166,424 @@ function openPatient(pid) {
...
-                "style='background-color: transparent' readonly /></td>\n";
-            $s .= " </tr>\n";
-        }
-    }
+<body>
+<div class="container mt-3">
+    <div class="row">
+        <div class="col-12">
+            <h2>
+                <?php echo ('Electronic Reports'); ?>
```
Comment (bradymiller) [inline interface/orders/list_reports.php:174]: This needs to be translated:
```
<?php echo xlt('Electronic Reports'); ?>
```

**Comment by stu01509** [comment]
Update the commit and remove an unnecessary file.

**Comment by bradymiller** [comment]
hi @stu01509 , Testing well and bringing this in.Thanks for the imrpovements! -brady

**Pull request merged**